docs/source/fr/quicktour.md

```diff
@@ -23,18 +23,20 @@ Soyez opérationnel avec 🤗 Transformers ! Que vous soyez un développeur ou u
 Avant de commencer, assurez-vous que vous avez installé toutes les bibliothèques nécessaires :
 
 ```bash
-!pip install transformers datasets
+pip install transformers datasets
```

Comment:
No the exclamation mark needs to stay for the notebook conversion of this doc page.